packages/client/src/encryption/EncryptionUtil.ts

```diff
@@ -139,7 +139,7 @@ class EncryptionUtilBase {
      * message content and returns null.
      */
 
-    static decryptStreamMessage(streamMessage: StreamMessage, groupKey: GroupKey) {
+    static decryptStreamMessage(streamMessage: StreamMessage, groupKey: GroupKey): any {
```

Comment:
```suggestion
    static decryptStreamMessage(streamMessage: StreamMessage, groupKey: GroupKey): null | never {
```

Reply:
The documentation seems to be out of date:

```
Decrypts the serialized content of 'streamMessage' with 'groupKey'. If the resulting plaintext is the concatenation
of a new group key and a message content, sets the content of 'streamMessage' with that message content and returns the key. If the resulting plaintext is only a message content, sets the content of 'streamMessage' with that
message content and returns null.
```

We don't seem to need the return value at all. Maybe this could return `void | never` (that it, it throws an `Error` or returns nothing)?